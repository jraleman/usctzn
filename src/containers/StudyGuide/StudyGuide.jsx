import React from 'react';
import styled from 'styled-components';
import PdfDownloader from '../../components/PdfDownloader';
import AudioPlayer from '../../components/AudioPlayer';

const studyGuide2020Pdf = '/assets/Study_Guide_-_Civic_Test_2020.pdf'
const questionsAnswersPdf = '/assets/128_Civics_Questions_and_Answers_(2020_version).pdf'

const StudyGuide = () => {
    const title = 'Study Guide';
    const pdfs = [
        {
            label: 'Study Guide 2020',
            src: studyGuide2020Pdf,
        },
        {
            label: '128 Civics Q&A',
            src: questionsAnswersPdf,
        }
    ];
    return (
        <StudyGuideContainer>
            <Title>{title}</Title>
            <AudioPlayer />
            <PdfDownloader pdfs={pdfs} />
        </StudyGuideContainer>
    );
};

const Title = styled.h2`
    text-align: center;
    text-transform: uppercase;
`;

const StudyGuideContainer = styled.div``;

export default StudyGuide;
