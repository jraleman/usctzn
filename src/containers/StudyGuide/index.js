import StudyGuide from './StudyGuide';

export default StudyGuide;
