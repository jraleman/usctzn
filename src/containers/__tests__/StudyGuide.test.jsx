import React from 'react';
import { render } from '@testing-library/react';
import StudyGuide from '../StudyGuide';

describe('StudyGuide', () => {
    it('renders', () => {
        const component = render(<StudyGuide />);
        expect(component).not.toBeNull();
        expect(component).toMatchSnapshot();
    });
});
