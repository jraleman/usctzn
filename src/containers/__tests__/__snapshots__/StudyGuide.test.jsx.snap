// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StudyGuide renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-hKwDye"
      >
        <h2
          class="sc-dkPtRN djehRW"
        >
          Study Guide
        </h2>
        <div>
          AudioPlayer
        </div>
        <div
          class="sc-gsDKAQ"
        >
          <a
            class="sc-bdvvtL MtoQy"
            download=""
            href="/assets/Study_Guide_-_Civic_Test_2020.pdf"
          >
            ⏬ 
            Study Guide 2020
             ⏬
          </a>
          <a
            class="sc-bdvvtL MtoQy"
            download=""
            href="/assets/128_Civics_Questions_and_Answers_(2020_version).pdf"
          >
            ⏬ 
            128 Civics Q&A
             ⏬
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-hKwDye"
    >
      <h2
        class="sc-dkPtRN djehRW"
      >
        Study Guide
      </h2>
      <div>
        AudioPlayer
      </div>
      <div
        class="sc-gsDKAQ"
      >
        <a
          class="sc-bdvvtL MtoQy"
          download=""
          href="/assets/Study_Guide_-_Civic_Test_2020.pdf"
        >
          ⏬ 
          Study Guide 2020
           ⏬
        </a>
        <a
          class="sc-bdvvtL MtoQy"
          download=""
          href="/assets/128_Civics_Questions_and_Answers_(2020_version).pdf"
        >
          ⏬ 
          128 Civics Q&A
           ⏬
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
